data/remnant/remnant 2 cognizance.txt

```diff
@@ -734,7 +734,7 @@ mission "Remnant: Cognizance 20"
 					goto missionchoice
 				`	"Why do you need multiple methods of decision making?"`
 			`	"Because we are in the midst of war, and located in an actively hazardous part of the galaxy. We, as a people, need to make countless thousands of on-the-spot decisions every day. If we had to reach a consensus for every troop movement, expedition, or resource allocation we would be overwhelmed within days." She stops speaking, and another Remnant picks up the explanation.`
-			`	"By the same token, prefects can be killed, or overloaded. Having as much decision making as possible handled by the Remnant as a group reduces our vulnerability to assassination strikes, and helps protect our prefects from burning out under the responsibility. On the flip side, it also protects our society from becoming an autocracy where one powerful person or small group makes all the decisions."`
+			`	"By the same token, prefects can be killed, or overloaded. Having as much decision making as possible handled by the Remnant as a group reduces our vulnerability to assassin strikes, and helps protect our prefects from burning out under the responsibility. On the flip side, it also protects our society from becoming an autocracy where one powerful person or small group makes all the decisions."`
```

Comment:
An assassin strike honestly sounds like something you'd get in Discworld when the assassins aren't getting paid enough. Maybe:
```suggestion
			`	"By the same token, prefects can be killed, or overloaded. Having as much decision making as possible handled by the Remnant as a group reduces our vulnerability to assassination, and helps protect our prefects from burning out under the responsibility. On the flip side, it also protects our society from becoming an autocracy where one powerful person or small group makes all the decisions."`
```

Reply:
Glad I'm not the only one to think that! I left it as assassin strike given the weirdness of Remnant vocabulary and speech patterns, it helps them feel more... off?

That being said, let me know if you want me to commit the suggestion and I'm happy to oblige

Reply:
If it's weirdness for the sake of intentional weirdness, then I think it's fine.

Reply:
Yea, the original here is better than the version in the PR.

This suggestion is, in turn, better than both, grammatically, but loses the linguist weirdness.

Reply:
If it is being included to highlight a linguistic quirk I would want to make it _more_ weird, so there is no doubt that it isn't just an error in editing. For example, in this context they're talking about a decapitation strike to take out the Remnant leadership, so they could use 'beheading strike' or 'guillotine strike'. The phrase could also be enclosed in quotes to show that it is not the narrator's choice of words.

Possibly the best way to do it would be the same as in some other missions, where it is described as a hand signal which is part-way between a couple of others, or that _gives the sense of_ decapitation (for example).

Reply:
I chose to use the phrasing `reduces our vulnerability to assassination strikes,` instead of `reduces our vulnerability to assassin strikes,` because I was referring to the type of strike, rather than the person doing it. This would be a similar meaning as talking about a decapitation strike or guillotine strike.

Reply:
So seems like there's 3 viable options:
1. Leaving this as-is (`assassination strikes`)
2. Changing to `assassination`
3. @LazerLit 's suggestion, which would be pretty neat but also require some more creativity.

@Zitchas which do you prefer?

Reply:
Hmmm, I think for the moment that simply leaving it as-is would be my preference. 

Creating an alternative "gesture that indicates a combination of meanings" is a possibility, but nor do I want to over-use that, either.